### ISSUE TITLE
Add arm64 .deb and binary builds

### ISSUE THREAD

**Issue by vielmetti** [spf13/hugo#3318]
Two target systems, really: a Raspberry Pi 3 running a Debian 8 based HypriotOS, and a Cavium ThunderX 96-core system from Packet (Type 2A) running Ubuntu 16.04 or CoreOS or CentOS 7.

#1772 is the only issue I can see that came up and was resolved, so this should be a small matter of changing "make build" to add one more target.

**Comment by vielmetti** [comment]
I followed the advanced build and install instructions and Hugo appears to be working flawlessly on arm64 when compiled on arm64, at least on the null site test.

**Comment by vielmetti** [comment]
`make test` passes:

```
root@docker-build-test:~/src/github.com/spf13/hugo# make test
govendor test +local  
?       github.com/spf13/hugo   [no test files]
ok      github.com/spf13/hugo/bufferpool        0.017s  
ok      github.com/spf13/hugo/commands  0.031s
?       github.com/spf13/hugo/config    [no test files]  
ok      github.com/spf13/hugo/create    0.112s
?       github.com/spf13/hugo/deps      [no test files]  
?       github.com/spf13/hugo/docshelper        [no test files]
ok      github.com/spf13/hugo/helpers   0.061s  
ok      github.com/spf13/hugo/hugofs    0.021s
ok      github.com/spf13/hugo/hugolib   1.774s  
ok      github.com/spf13/hugo/i18n      0.026s
?       github.com/spf13/hugo/livereload        [no test files]  
ok      github.com/spf13/hugo/media     0.017s
ok      github.com/spf13/hugo/output    0.025s  
ok      github.com/spf13/hugo/parser    0.021s
ok      github.com/spf13/hugo/source    0.023s
?       github.com/spf13/hugo/tpl       [no test files]  
ok      github.com/spf13/hugo/tpl/tplimpl       2.090s
ok      github.com/spf13/hugo/transform 0.022s
?       github.com/spf13/hugo/utils     [no test files]
?       github.com/spf13/hugo/watcher   [no test files]
```

**Comment by bep** [comment]
This will be fixed by #3358
